### ISSUE TITLE
proposal: `<comment_all_ignores>`

### ISSUE THREAD

**Issue by Number-3434** [dart-lang/linter#4860]
## <lint_name>

*Replace <lint_name> above with your proposed rule name.  (See notes on naming in [Writing Lints].)*

## Description

*Provide a short description suitable for display in console output.*

## Details

*Give a detailed description that should provide context and motivation.  Ideally this could be used directly in the rule's documentation.*

## Kind

*Does this enforce style advice?  Guard against errors?  Other?*

## Bad Examples

*Add a few examples that demonstrate where this lint should fire.*

## Good Examples 

*Add a few examples that demonstrate a “good” adoption of this lint’s principle.*

## Discussion

*Add any other motivation or useful context here.*

### Discussion checklist

- [ ] List any existing rules this proposal modifies, complements, overlaps or conflicts with.
- [ ] List any relevant issues (reported here, the [SDK Tracker], or elsewhere).
- [ ] If there's any prior art (e.g., in other linters), please add references here.
- [ ] If this proposal corresponds to [Effective Dart] or [Flutter Style Guide] advice, please call it out. (If there isn't any corresponding advice, should there be?)
- [ ] If this proposal is motivated by real-world examples, please provide as many details as you can.  Demonstrating potential impact is _especially valuable_.

[Writing Lints]: https://github.com/dart-lang/linter/blob/main/doc/writing-lints.md
[Effective Dart]: https://dart.dev/effective-dart
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[SDK Tracker]: https://github.com/dart-lang/sdk/issues

**Comment by srawlins** [comment]
I'd love this rule. I miss it often, from other linters, haha.

**Comment by Number-3434** [comment]
> I'd love this rule. I miss it often, from other linters, haha.

Yeah, also you would have to add an ignore comment to ignore this rule :)

```dart
ignore: comment_all_ignores, because it is dumb
ignore: some_lint
```

**Comment by pq** [comment]
@goderbauer: I'm guessing this'd be used *at least* for the flutter repo and maybe flutter recommended style?

@bwilkerson, @scheglov, @srawlins, @keertip, @kallentu: I wonder if this is something we'd consider for analyzer?

**Comment by goderbauer** [comment]
I would love to enforce this for the flutter repo.

Not so sure about putting it in the flutter recommended style set, though, since it isn't really a lint specific to flutter code. Since it is about general dart code, if we think this is useful we should align across the dart universe and put it in the general recommended set.

**Comment by srawlins** [comment]
Implemented in https://github.com/dart-lang/sdk/commit/607b3f5ddba34348f965cce99e4cac5e87a46a99

**Comment by Number-3434** [comment]
yay!
